tests/ci/profile_handler.go

```diff
@@ -393,22 +394,25 @@ func GenerateClusterCreationArgsByProfile(token string, profile *Profile) (clust
 		// ToDo
 	}
 
-	if profile.ClusterName != "" {
-		clusterArgs.ClusterName = profile.ClusterName
+	if constants.RHCS.RHCSClusterName != "" {
```

Comment:
I added the support of that variable so that, if defined, you can reapply the different TF files with still the same prefix. Pretty useful when working on day1 negative :)